tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "lib": [
       "dom",
       "dom.iterable",
-      "esnext"
+      "esnext",    
```

Comment:
please, always check also white spaces
by doing this we can prevent merge conflicts in future